### PR TITLE
Devdocs 4182 - Orders V3

### PR DESCRIPTION
**Pull request by bc-tgomez**
# [DEVDOCS-4182](https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4182)

## Orders v3 YAML malformed 503 response description
* Correct 503 response in Orders v3

### CODE REVIEW

**Review by traci707 — COMMENTED** [review]
Looks good, I don't see an error for the 503 response but there is an error further down on line 733.  I don't know if it is related or not.

**Review by traci707 — DISMISSED** [review]

File: reference/orders.v3.yml [hunk 1]
```diff
@@ -712,7 +712,7 @@ paths:
                         }
                     }
         '503':
-          description: All failure response.
+          description: "All failure response. \nHTTP status code for the entire batch request to roll up to the most severe individual failure to the whole request."
```
Comment (slsriehl) [inline reference/orders.v3.yml:715]:
```suggestion
          description: "All <<what?>> failed. \nThe HTTP status code for the entire batch request rolls the most severe individual failure up to the whole request."
```
I'm not sure what this means. Rolls up? is this a directional/spatial metaphor for data/error flow?

**Review by slsriehl — CHANGES_REQUESTED** [review]
see comment.

**Comment by bc-tgomez** [comment]
> 

Many thanks @slsriehl for the feedback!

**Review by slsriehl — APPROVED** [review]
excellent.

**Pull request merged**